_posts/2016-12-06-the-state-of-babel.md

```diff
@@ -0,0 +1,460 @@
+- In September, we released the first version of [babel-preset-env](https://github.com/babel/babel-preset-env).
+- After a year on Phabricator, we moved back to to GitHub issues thanks solely to [@danez](https://github.com/danez) and his amazing (and underappreciated) work.
+
+Now `babel-core` is downloaded over 5 million times per month and almost 60 million times total.
+
+Thanks everyone for your continued support! We want to continue acting as the foundation of the JavaScript toolchain: compilation, linting, minification, codemods, code coverage, etc.
+
+## Current Status
+
+We understand that working on any of these could be its own project. If you're interested in helping out please check out the issues linked below!
+
+### Maintaining Babel plugins [for each proposal in TC39](https://github.com/tc39/proposals) starting from Stage 0
+
+Babel uses [TC39's concept of stages](http://babeljs.io/docs/plugins/#stage-x-experimental-presets-) to categorize its experimental plugins. Users should be able to easily use features before they are all implemented in browsers in [stage 4 of the TC39 process](https://tc39.github.io/process-document/).
+
+Babel is fundamental in the process given its place in the ecosystem: it is significantly easier for a developer to update a `.babelrc` file than a browser flag and much faster to write a Babel plugin than to implement the feature natively in the browser. This is the core of Babel.
+
+But the proposals process involves significant iteration: proposals can change in syntax or even get dropped. Because TC39 meets every 2 months, plugins should be updated as least as often as each change in stage so that users can be in sync.
+
+> Early feedback to the proposal champion and committee is extremely valuable although it is recommended to use Stage 0/1/2 features with caution.
+> Although companies like Facebook take advantage of these kinds of features, they have created codemods to allow easy change.
+
+---
+
+There isn't enough time or resources to maintain each plugin, especially when there are spec updates.
+
+- Some transforms are simply out of date such as decorators. Logan had to port the previous spec [babel-plugin-transform-decorators-legacy](https://github.com/loganfsmyth/babel-plugin-transform-decorators-legacy) for Babel 6 and we haven't had anyone able to rewrite it for the updated spec.
+- [babel/babel#3473 - Async iteration proposal](https://github.com/babel/babel/pull/3473) wasn't merged for so long because we just didn't have time to review it. By the time it was merged it had already moved 
+from stage 2 to stage 3.
+
+---
+
+Next we're going to be working with [Daniel Ehrenberg](https://github.com/littledan) on [Private Fields (Stage 2)](https://github.com/tc39/proposal-private-fields) and [Yehuda Katz](https://github.com/wycats) on [Class and Property Decorators (Stage 2)](http://tc39.github.io/proposal-decorators/).
```

Comment:
Feel free to add this (but don’t feel obligated):

```
…and [Mathias Bynens](https://mathiasbynens.be/) on [Unicode Property Escapes in Regular Expressions (Stage 2)](https://github.com/tc39/proposal-regexp-unicode-property-escapes)
```

There has already been work in this area, i.e. #3683.